src/DashJS/SDK/SDK.ts

```diff
@@ -0,0 +1,37 @@
+// import {Wallet} from "@dashevo/wallet-lib";
+import {Wallet} from "../../../../wallet-lib/src";
+import {Platform} from '../Platform';
+// @ts-ignore
+import DAPIClient from "@dashevo/dapi-client"
+// @ts-ignore
+
+const defaultSeeds =  [
+    '18.237.69.61',
+    '18.236.234.255',
+].map(ip => ({ service: `${ip}:3000` }));
+
+export class SDK {
+    private network: string | string;
+    public wallet: Wallet | undefined;
+    public platform: Platform | undefined;
+    public client: DAPIClient;
+
+    constructor(opts: {
+        schema?: object;
+        network: string;
+        mnemonic?: string
+    }) {
+        this.network = opts.network;
+
+        this.client = new DAPIClient(Object.assign({seeds:defaultSeeds,
+            timeout: 20000,
+            retries: 15},opts));
+        if(opts.mnemonic) {
+            // @ts-ignore
+            this.wallet = new Wallet(opts);
```

Comment:
Well : 

- We want to provide ability to fetchDocuments, which only requires a schema at init and later at call of the function the type/opts of the query. 
- We want to provide ability pushDocuments which will requires above schema, but also the document. In order to broadcast it, we will need to sign it. So that's where Wallet-lib will be needed. 
So for now, especially due to the fact that we only provide fetch for now, I think we should allow ability to not init wallet (which can be quite expensive ressource-wise / size-wise)